accounts/views.py

```diff
@@ -1,3 +1,64 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from accounts.forms import SignUpForm
+from django.contrib.auth import login as auth_login
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from accounts.models import Account
+from accounts.serializers import AccountSerializers
 
-# Create your views here.
+
+def index(request):
+    return render(request, 'accounts/base.html')
+
+
+def signup(request):
+    if request.method == 'POST':
+        form = SignUpForm(request.POST)
+        if form.is_valid():
+            user = form.save()
+            auth_login(request, user)
+            return redirect('accounts:index')
+
+    else:
+        form = SignUpForm()
+
+    return render(request, 'accounts/signup.html', {'form': form})
+
+
+class accountListView(generics.ListAPIView):
```

Comment:
1. Class names in py are always `CamelCase`
2. Instead of creating separate views for list, create, etc use a single class inheriting from `ModelViewSet`

Reply:
on it sir

Reply:
@chauhanprakhar @Atishay990 
look into it